Main_Solution/bonus_task_test.rb

```diff
@@ -16,45 +19,40 @@ def setup
   end
 
   def test_hover
-
     link_to_profile = hover_on_avatar_of_user 1
-    assert(link_to_profile.displayed?)
+    expect(link_to_profile.displayed?).to be
 
     link_to_profile = hover_on_avatar_of_user 2
-    assert(link_to_profile.displayed?)
+    expect(link_to_profile.displayed?).to be
 
     link_to_profile = hover_on_avatar_of_user 3
-    assert(link_to_profile.displayed?)
-
+    expect(link_to_profile.displayed?).to be
   end
 
   def test_drag_and_drop
     navigate_to 'https://the-internet.herokuapp.com/drag_and_drop'
 
     source_element_id = '#column-a'
     target_element_id = '#column-b'
-
     columns_hash = html5_drag_and_drop source_element_id, target_element_id
 
-    assert_equal('B', columns_hash[:source_element].text)
-    assert_equal('A', columns_hash[:target_element].text)
+    expect(columns_hash[:source_element].text).to eql 'B'
+    expect(columns_hash[:target_element].text).to eql 'A'
 
   end
 
   def test_select_from_list
-
     navigate_to 'https://the-internet.herokuapp.com/dropdown'
     dropdown_id = 'dropdown'
     available_options = ['1', '2']
 
     available_options.each do |option|
-      assert_equal(option, select_option_from_dropdown(dropdown_id, option))
+      expect(select_option_from_dropdown(dropdown_id,option)).to eql option
```

Comment:
if you're doing a lot of assertions in one place it would be better to wrap them in the ```aggregate_failures``` block
http://www.relishapp.com/rspec/rspec-expectations/v/3-5/docs/aggregating-failures